ansible/ansible-version.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: all
```

Comment:
> If hosts: localhost is used instead then the version is not actually enforced if localhost is not part of play.

How do you call this? Using `hosts: localhost` is how it worked for me while I was testing the assert task.